egs/wsj/s5/utils/data/get_utt2num_frames.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+
```

Comment:
this script doesn't seem to be used.  of course it might be useful in future... but maybe best not to include it for now, if not needed.